fmn/cache/util.py

```diff
@@ -25,7 +26,7 @@ def cache_arg(arg: str, scope: str | None = None) -> Callable[[str, str | None],
 
     @ft_cache
     def get_cache_arg(*cached_fn_args, **cached_fn_kwargs) -> Any:
-        f"""Get the cache arg {arg} for scope {scope}."""
+        """Get the cache arg {arg} for scope {scope}."""
```

Comment:
Hmm, we should probably remove this… the return value is a function but it is directly passed onto cashews which probably doesn’t need the docstring.

Reply:
The rule that picked it up said that the f-string mark would prevent Python from recognizing it as a docstring. Aside from that I'm fine with keeping it as a plain string if it's useful doc for humans, or removing it if not.

Reply:
> The rule that picked it up said that the f-string mark would prevent Python from recognizing it as a docstring.

Ah, wasn’t aware of that. It was supposed to contain the actual values for `arg` and `scope`, as the function is dynamically generated. I’d say, just remove it.